scripts/search_start_points.py

```diff
@@ -0,0 +1,68 @@
+from itertools import product
+from scipy.optimize import minimize
+
+import numpy as np
+import os
+import csv
+
+from scripts.models import Qlearning, RescorlaWagner
+from scripts.player import Estimator, RealPlayer
+
+
+def save_optimal_params(data_dir_path, model, new_filename):
+    all_filenames = os.listdir(data_dir_path)
+    with open('{}.csv'.format(new_filename), 'w') as file:
+        writer = csv.writer(file, delimiter=';')
+        writer.writerow(get_header(model))
+        for filename in all_filenames:
+            if filename.endswith('xls'):
+                row = []
+                rp = RealPlayer(os.path.join(data_dir_path, filename))
+                estimator = Estimator(decisions=rp.data['Action'].tolist(),
+                                      condition_left=rp.data['StimulusLeft'].tolist(),
+                                      condition_right=rp.data['StimulusRight'].tolist(),
+                                      rewards=rp.data['Reward'].tolist(),
+                                      model=model)
+                player_name = os.path.splitext(os.path.basename(filename))[0][:-8]
+                player_parameters = optimal_parameters(estimator, model)
+                row.append(player_name)
+                row.extend(player_parameters)
+                writer.writerow(row)
+
+
+def optimal_parameters(estimator, model):
```

Comment:
we'll get mixed with this cause these are optimal starting points and not optimal parameters

Reply:
you dont have to use model as param as it is a field of estimator

Reply:
and the best idea is to make this a method of RealPlayer